holidayextras/holidayextras.github.com
# Adding calculation for estimated reading time

#### What does this PR do?
- Removes the need to a manual estimation on how long it will take to read a particular blog post.
- Estimates the reading time based on words on the page at a reading speed of 180 words per minute (wpm)
- Removes all manual entires

#### How should a developer review this?
- checkout branch
- run `jekyll serve --drafts` this should boot up the blog locally
- Ensure a reading time is displayed on each blog post
- Check the code most of it is deleting lines on old times from old blog posts

#### How should this be manually tested?
- I'm not sure its worth checking the reading time is exact to the nearest second, maybe just check its not saying a 30min read for a 4 line blog post

#### Screenshots (if appropriate)
![screen shot 2015-01-15 at 15 15 35](https://cloud.githubusercontent.com/assets/778942/5759949/643ff272-9cc9-11e4-9e26-fbfba091677f.png)

#### What gif best describes how you feel about this work?
![reading](http://media.giphy.com/media/34ZNcoaN5u4hi/giphy.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
